src/lib/containers/Error.tsx

```diff
@@ -14,9 +56,16 @@ export const Error = (props: ErrorProps) => {
     return <></>
   }
 
-  const loginError = error.status === 403 && !token
-  const accessDenied = error.status === 403 && token
-
+  let synapseClientError: SynapseClientError | undefined = undefined
+  let typeError: TypeError | undefined = undefined
```

Comment:
Could this TypeError be expanded to the Error type, and in any case grab the associated message?  It would be a shame to need special cases for all of the standard error types (URIError, SyntaxError, InternalError, EvalError, ...)